src/main/windows.ts

```diff
@@ -101,19 +106,34 @@ export const openAppWindow = (app: LaunchableApp) => {
         }
     }
 
-    const appWindow = browser.createWindow({
-        title: `${app.displayName || app.name} v${app.currentVersion}`,
-        url: `file://${getElectronResourcesDir()}/app.html?appPath=${app.path}`,
-        icon: getAppIcon(app),
-        x,
-        y,
-        width,
-        height,
-        minHeight: 500,
-        minWidth: 760,
-        show: true,
-        backgroundColor: '#fff',
-    });
+    const { device } = openAppOptions;
+    const additionalArguments: string[] = [];
+    if (device != null) {
```

Comment:
```suggestion
    if (device !== null) {
```

I believe such notation is prefered, also I'd check just `if (device)` bcz who knows what may be returned in future (maybe null will be replaced with undefined or empty string)

Reply:
I know different people have different opinion on this matter, but if you only check for `null` strictly, you'd have to add a check of `undefined` as well. In this case we want to have the same behavior both if device is `null` or `undefined`, and then it should be ok to check for `null` loosely, because it will check for `undefined` and `null`.